org.eclipse.persistence.asm/src/main/resources/about.html

```diff
@@ -16,10 +16,10 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"/>
-<title>About ASM ${asm.version}</title>
+<title>About ASM v9.1.0</title>
```

Comment:
Is this change intentional? maven can replace the property with the real value for us during the build, so we don't have to worry about keeping the version number in the file up-to-date ourselves

Reply:
I had made this change because I wasn't seeing the variable get replaced when I ran a build in eclipselink/master, but now it's working so i dont know... I'll put it back

Reply:
try in eclipselink repo:

```
cd plugins/org.eclipse.persistence.asm
mvn -U -C clean install -Poss-release
```

that's how latest (9.1.0) version was produced. I expect that when I clone this repo and do:

```
cd org.eclipse.persistence.asm
mvn -U -C clean install -Poss-release
```

I get the same result (but for sth like 9.2.0-SNAPSHOT)

Reply:
> mvn -U -C clean install -Poss-release
> [WARNING] The requested profile "oss-release" could not be activated because it does not exist.
I can see no reference to this profile in eclipselink/master/plugins/org.eclipse.persistence.asm. Is this right?

My validation is that I execute `mvn -U -C clean install` and then run a compare against https://mvnrepository.com/artifact/org.eclipse.persistence/org.eclipse.persistence.asm/9.1.0